packages/primitives/button/test/index.ts

```diff
@@ -0,0 +1,44 @@
+import test from 'tape'
+import { renderApp } from '../../shared/test-utils'
+
+type TFixtures = 'bare-bones' | 'disabled'
+const renderButton = (filename: TFixtures) => renderApp(require.resolve(`./fixtures/${filename}.tsx`))
+// const a11yReport = (filename: TFixtures) => getA11yReport(require.resolve(`./fixtures/${filename}.tsx`))
+
+const PACKAGE = '@primitives/button'
+
+test(`${PACKAGE} has proper role`, async (t) => {
+  const page = await renderButton('bare-bones')
+  const btnEl = await page.$('button')
+  const a11ySnapBtn = await page.accessibility.snapshot({ interestingOnly: false, root: btnEl })
+  // Waiting for a fix on r11y side so we can refactor this test
+  // https://github.com/nextools/metarepo/issues/233
+  // const report = await a11yReport('bare-bones')
+
+  t.equal(a11ySnapBtn.role, 'button', 'button role is `button`')
+
+  await page.close()
+})
+
+test(`${PACKAGE} can be focused`, async (t) => {
+  const page = await renderButton('bare-bones')
+  const btnEl = await page.$('button')
+
+  await page.focus('button')
+
+  const a11ySnap = await page.accessibility.snapshot({ root: btnEl })
+
+  t.ok(a11ySnap.focused)
+  await page.close()
+})
+
+test(`${PACKAGE} can be disabled`, async (t) => {
```

Comment:
can we actually check that it is not clickable? or the `disabled` status is checking it?

Reply:
disabled buttons don't fire events

Reply:
But it would be possible to assert if a function has been called, right? 
Maybe for primivites it's unnecessary but for molecules i think it does

Reply:
when you need, you can add a module to handle mocking method calls, [this post](https://ponyfoo.com/articles/testing-javascript-modules-with-tape#leveraging-proxyquire-to-mock-modules) has an example with `proxyrequire`